src/java.naming/share/classes/javax/naming/event/package-info.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 1999, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ *
+ * Provides support for event notification when accessing naming and
+ * directory services.
+ *
+ * <p>
+ * This package defines the event notification operations of the Java Naming
+ * and Directory Interface (JNDI). &nbsp;
+ * JNDI provides naming and directory functionality to applications
+ * written in the Java programming language. It is designed to be
+ * independent of any specific naming or directory service
+ * implementation. Thus a variety of services--new, emerging, and
+ * already deployed ones--can be accessed in a common way.
+ *
+ * <h2>Naming Events</h2>
+ * <p>
+ * This package defines a {@code NamingEvent} class to represent an event
+ * that is generated by a naming/directory service.
+ * It also defines subinterfaces of {@code Context} and {@code DirContext},
+ * called {@code EventContext} and {@code EventDirContext},
+ * through which applications can register their interest in events
+ * fired by the context.
+ * <p>
+ * {@code NamingEvent} represents an event that occurs in a
+ * naming or directory service. There are two categories of naming events:
+ * <ul>
+ * <li>Those that affect the namespace (add/remove/rename an object)
+ * <li>Those that affect the objects' contents.
+ * </ul>
+ * Each category of events is handled by a corresponding listener:
+ * {@code NamespaceChangeListener}, {@code ObjectChangeListener}.
+ * <p>
+ * An application, for example, can register its interest in changes to
+ * objects in a context as follows:
+ * <blockquote>
```

Comment:
The leftover `<blockquote>` can be removed:
```suggestion
```

Reply:
Fixed both in [657ef5c](https://github.com/openjdk/jdk/pull/19529/commits/657ef5c7532bc587cdead80d35486f30bb931d5e), thank you.